src/MembershipsSDKAdapter.js

```diff
@@ -0,0 +1,110 @@
+import {BehaviorSubject, throwError} from 'rxjs';
+import {DestinationType, MembershipsAdapter} from '@webex/component-adapter-interfaces';
+
+export default class MembershipsSDKAdapter extends MembershipsAdapter {
+  /**
+   * Gets the active members in a meeting
+   *
+   * @static
+   * @private
+   * @param {object} members Members object from meeting, keyed by ID
+   * @returns {Array}
+   * @memberof MembershipsSDKAdapter
+   */
+  static getActiveMembers(members) {
+    return Object.values(members)
+      .filter((member) => !member.isInMeeting)
+      .map((member) => ({
+        id: member.id,
+      }));
+  }
+
+  constructor(datasource) {
+    super(datasource);
+
+    this.membershipSubjects = {};
+  }
+
+  /**
+   * Gets a shared rxjs subject for the given membership
+   *
+   * @private
+   * @param {string} membershipID  ID of the destination for which to get members
+   * @param {DestinationType} destinationType  type of the membership destination
+   * @returns {BehaviorSubject}
+   */
+  getOrCreateMembershipSubject(membershipID) {
```

Comment:
hahaha very Java-esc name, maybe simplifying it to "get" should be sufficient.
I suggest we can expand on the docstring that it will be created if it doesn't exist
```suggestion
  getMembershipSubject(membershipID) {
```

Reply:
The name works fine as is.